v2.0/foreign-key.md

```diff
@@ -49,6 +49,18 @@ For example, if you have a Foreign Key constraint on columns `(A, B)` and try to
 
 However, allowing _NULL_ values in either your foreign key or referenced columns can degrade their referential integrity. To avoid this, you can use the [Not Null constraint](not-null.html) on both sets of columns when [creating your tables](create-table.html). (The Not Null constraint cannot be added to existing tables.)
 
+### Foreign Key Actions
+<span class="version-tag">New in v2.0:</span> You can specify a foreign key reference action.
+
+Parameter | Description
+----------|------------
+`ON DELETE NO ACTION` / `ON UPDATE NO ACTION` | _Default actions._ Fail if there are any existing references to the key being updated or deleted, but allow this check to be deferred until the end of the statement. <br><br>Aliases: `ON DELETE RESTRICT` / `ON UPDATE RESTRICT`
+`ON DELETE RESTRICT` / `ON UPDATE RESTRICT` | `RESTRICT` and `NO ACTION` are currently equivalent until options for deferring constraint checking are added. To set an existing foreign key action to `RESTRICT`, the foreign key constraint must be dropped and recreated.
+`ON DELETE CASCADE` / `ON UPDATE CASCADE` | When a referenced foreign key is deleted or updated, all rows referencing that key are deleted or updated. If there are other alterations to the row, such as a `SET NULL` or `SET DEFAULT`, the delete with take precedence. <br><br>Note that `CASCADE` does not list objects it drops or updates, so it should be used cautiously.
```

Comment:
"the delete with take precedence" seems incomplete or missing code formatting.

Reply:
Typo - fixed!